x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/esql/search_filter.cy.ts

```diff
@@ -43,51 +44,51 @@ describe.skip(
     beforeEach(() => {
       login();
```

Comment:
might not be necessary, but wouldn't we want to delete all created timelines before doing anything, using this [deleteTimelines](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/tasks/api_calls/common.ts#L116) function?

Reply:
Yea, I introduced it, thanks! It may not be super necessary since we create a new timeline instance before testing, but generally a good idea to clean up